.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
```

Comment:
Shouldn't the event be the same as the other CI jobs?

https://github.com/GoogleChrome/chromium-dashboard/blob/fbcfef6e873d59a3bb91cb5172821912b6cdfa5e/.github/workflows/ci.yml#L8-L12

Reply:
I copied the initial playwright.yml from the bikeshed version.  So, I don't know what "dependabot/**" and "daily-test-sync" are about, and you may well be right.  I'll change it and see what happes.